### PR TITLE
added 0123-ssd-mobilenet-v2-0.35.1.-barrier-256x256 model

### PR DESCRIPTION
**Pull request by elenash**

### CODE REVIEW

**Comment by elenash** [comment]
@snosov1 Can you take a look?

**Comment by atsyplen** [comment]
Jenkins please retry a build

File: model_downloader/list_topologies.yml [hunk 1]
```diff
@@ -1097,6 +1097,29 @@ topologies:
     framework: mxnet
     license: https://github.com/lachinov/brats2018-graphlabunn/blob/master/LICENSE
 
+  - name: "ssd-mobilenet-v2-0.35.1.-barrier-256x256-0123"
```
Comment (snosov1) [inline model_downloader/list_topologies.yml]: Let's use the name `vehicle-license-plate-detection-barrier-0123"

No need to change the name of the tarball.

**Comment by kchechil** [comment]
Jenkins please retry a build

**Comment by snosov1** [comment]
@kchechil are we good to merge?

**Comment by kchechil** [comment]
@snosov1 this model is not in OMZ roadmap, I do not see it in  benchmark, accuracy validation (who is doing that?), accuracy metrics, documentation, updated/checked demos etc. So it's up to you.

**Comment by elenash** [comment]
@kchechil It is in benchmark and its config is also added to accuracy validation (but not currently being validated as for other public models), demo also successfully runs using this model.

**Comment by kchechil** [comment]
@elenash  I can't match between different names in benchmark, name here and name in OMZ roadmap.
Can you please provide proof links?
Anyway, it looks like you said you did all the steps including validation yourself, so @snosov1 - as I said, it is up to you.

**Comment by elenash** [comment]
@kchechil The final name of this model is vehicle-license-plate-detection-barrier-0123 ( it is the same here, in benchmark and in accuracy checker). The commit description here contains incorrect model name (the first version) so it leads to the confusion. I'll send you links by email

**Comment by kchechil** [comment]
@elenash  is it actual? Could you please re-target and update to opencv:develop folders structure if yes? then it can be considered in Q4 scope.

**Comment by vladimir-dudnik** [comment]
@elenash we will close this PR due to lack of activity in support

**Comment by IRDonch** [comment]
> @elenash we will close this PR due to lack of activity in support

Closing as promised. If you're still interested in adding this model, please rebase your branch to `develop`, reopen the PR and retarget it to `develop` as well.

**Pull request closed without merging**